app/src/main/java/net/gsantner/opoc/ui/FilesystemViewerFragment.java

```diff
@@ -477,6 +478,15 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 }
                 return true;
             }
+
+            case R.id.action_copy: {
+                if (_filesystemViewerAdapter.areItemsSelected()) {
+                    Runnable copy = () -> WrMarkorSingleton.getInstance().copyContentOfFile(_filesystemViewerAdapter.descriptionFile(), getContext());
```

Comment:
* getSelection.get(0) for getting file
* ` markor/app/src/main/java/net/gsantner/opoc/util/ShareUtil.java ::setClipboard(text)` for setting clipboard
* Fileutils for reading the file

Reply:
Fix in wladokacur/markor@6c6f39893eb4315466faf99062318f8da95c45a8